src/routes/products/data.js

```diff
@@ -0,0 +1,114 @@
+import sardinoBlanco from '$lib/assets/bottles/sardino-blanco-big.avif'
+import sardinoRojo from '$lib/assets/bottles/sardino-rojo-big.avif'
+import forzudoRojo from '$lib/assets/bottles/forzudo-rojo-big.webp'
+import forzudoBlanco from '$lib/assets/bottles/Frozudo-Blanco_med_skygge_900x.webp'
```

Comment:
why this new weird naming and webp instead of .avif ?

Reply:
They are the same images of the official page forzudo the big image is webp